<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyTemplateRuleForm</class>
 <widget class="GUIFormBase" name="PropertyTemplateRuleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetSelectLib">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>400</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>全て</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditItemName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>選択するアイテム名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>設定しなければ領域内の全てのアイテムを選択</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ライブラリで選択</string>
    </attribute>
    <widget class="QLabel" name="labelLibFolderName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ＸＸＸ</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetGeneratedLibList">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>今回使用する設定</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetLibList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButtonEditLibFolder">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>分類選択</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Qtres/Class.png</normaloff>:/Qtres/Class.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>登録されているライブラリリスト</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSetFrom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>39</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>=&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGetBackAll">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>39</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSetFromAll">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>39</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>=&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGetBack">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>39</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;=</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidgetAlgorithmType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>TypeID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TypeName</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>アイテムリスト</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetItemList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Phase</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Layer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
